zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -290,8 +290,15 @@ pub(crate) fn transparent_output_address(
     output: &transparent::Output,
     network: Network,
 ) -> Option<transparent::Address> {
-    let script = zcash_primitives::legacy::Script::from(&output.lock_script);
-    let alt_addr = script.address();
+    // Create a `TxOut` primitive from a zebra `Output`.
+    let tx_out = zcash_primitives::transaction::components::TxOut {
```

Comment:
We could move this code to an `Output` method `recipient_address()` if we want.

Reply:
I'd say that would be handy.

Reply:
We're trying to keep everything that calls `zcash_primitives` directly in this module, so it's easy to see what we're using.

But we could add a convenience method to `Output`, like we did for `AuthDigest` and `Transaction`.